src/ctim/schemas/ttp.clj

```diff
@@ -106,12 +106,11 @@
                     (str
                      "specifies other TTPs asserted to be related to this cyber"
                      " threat TTP"))
-     :source (describe s/Str "source of this cyber threat TTP")})
+     :source (describe s/Str "source of this cyber threat TTP")
+     :kill_chains [v/KillChain]})
```

Comment:
This Kill chain phase should be associated with Indicators will not be appropriate at TTP level